examples/src/legacy/custom_cmm_example.py

```diff
@@ -0,0 +1,87 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
This is all good. You don't need to change this example with my comments on the MPL example.